src/lib/y2network/dialogs/add_interface.rb

```diff
@@ -32,11 +32,22 @@ module Dialogs
     class AddInterface < CWM::Dialog
       def initialize(default: nil)
         @type_widget = Widgets::InterfaceType.new(default: default ? default.short_name : nil)
+
+        textdomain "network"
+      end
+
+      def title
+        # Translators: Add interface dialog label
```

Comment:
More context for translators:

```suggestion
        # Translators: Dialog headline: adding a configuration of a network interface
```